### ISSUE TITLE
404 for https://swiftpackageindex.com/fummicc1/EasyFirebaseSwift

### ISSUE THREAD

**Issue by finestructure** [SwiftPackageIndex/SwiftPackageIndex-Server#1492]
recently added, doesn’t seem to be private or renamed

https://swiftpackageindex.com/fummicc1/EasyFirebaseSwift

**Comment by finestructure** [comment]
It's loading correctly on staging

https://staging.swiftpackageindex.com/fummicc1/EasyFirebaseSwift

**Comment by finestructure** [comment]
Ok, this has auto-corrected itself after re-analysis. The default branch was changed and the issue was that the version hadn't yet caught up. I.e. we had no `latest == default_branch` version, because the ref was still `.branch(develop)` while `repositories` was already on `default_branch == main`.

We could consider somehow shortening the window during which this is exposed but a) it's only an issue for a short while in the rare occasion that a default branch is changed and b) any fix for this would probably be rather complicated as we'd need to somehow communicate across processing stages.

BTW, I'm pretty sure we wouldn't have shown the package in the "recently added" list (where I found it) in the past, because it'd have no package name (because it has no default version). I feel like this is somewhat preferable over showing it but 404ing.
